deploy/index.html

```diff
@@ -292,6 +300,11 @@ <h3 id="sharing-user-known" class="d-none">
 <script src="modules/intro.js/intro.min.js"></script>
 <script src="modules/reparse.js"></script>
 <script src="modules/boolean-expression.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.min.js"></script>
```

Comment:
All other modules are kept locally. Even thought it's a small file, please create a local `modules/es6-promise` folder for it and import it locally.

Reply:
Oh sorry it's my fault, forgot to download it.